polaris/polaris/management/commands/watch_transactions.py

```diff
@@ -119,10 +119,9 @@ def process_response(cls, response):
     def match_transaction(cls, response: Dict, transaction: Transaction) -> bool:
         """
         Determines whether or not the given ``response`` represents the given
-        ``transaction``. Polaris does this by constructing the transaction memo
-        from the transaction ID passed in the initial withdrawal request to
-        ``/transactions/withdraw/interactive``. To be sure, we also check for
-        ``transaction``'s payment operation in ``response``.
+        ``transaction``. Polaris does this by checking the 'memo' field in the horizon
+        response matches the `transaction.memo` if present, as well as ensuring the
+        transaction includes a payment operation of the anchored asset.
```

Comment:
We should add some explanation of what happens if the transaction.memo isn't present.  It matches that the sender is the account authed by SEP10 instead right?  Does it require that check if the memo is present?

Reply:
It makes sure the source of the payment op matches the stellar account on the transaction saved to the DB, so yes. And its not an either-or situation, it does both. If the memo isn't present is just compares using all the other conditions like payment amount, account, etc.